src/tox/package.py

```diff
@@ -183,7 +183,12 @@ def perform_isolated_build(build_info, package_venv, session, config):
                 build_info.backend_module, build_info.backend_object, "sdist", str(config.distdir)
             )
         )
+
+        # need to start with an empty (but existing) source distribution folder
+        if config.distdir.exists():
+            config.distdir.remove(rec=1, ignore_errors=True)
```

Comment:
potential race here (exists check finishes, other process creates), though I suspect it doesn't matter -- though could this just be `config.distdir.remove(rec=1, ignore_errors=True)`?

Reply:
that was my first solution, until the test started failing; seems rec=1 is only active if the folder exists because of ``https://github.com/pytest-dev/py/blob/master/py/_path/local.py#L204``. Long story short you can only delete if already exists...